cultivation/scripts/task_management/session_timer.py

```
@@ -0,0 +1,249 @@
+# --- Prompt for session info ---
+def prompt(msg, default=None):
+    if default is not None:
+        return input(f"{msg} [{default}]: ") or default
+    return input(f"{msg}: ")
+
+import os
+
+# The log file is now stored in the repo by default; override with LOG_FILE env var for testing or custom setups.
+LOG_FILE = os.environ.get("LOG_FILE", DEFAULT_LOG_FILE)
+
+def log_session(task_id_display, start_time, end_time, duration_td, note_str, status, log_file=None):
```

Pylint (3.3.7)

[convention] Line too long (101/100)

(C0301)